### PR TITLE
mobile ui better

### PR DESCRIPTION
**Pull request by srt10coupe**
example: log and warnings views

### CODE REVIEW

**Comment by srt10coupe** [comment]
my state that is this contribution MIT licensed.

**Comment by LukasReschke** [comment]
@jancborchardt

**Comment by LukasReschke** [comment]
@owncloud-bot This is okay to test.

File: core/css/mobile.css [hunk 1]
```diff
@@ -21,6 +30,35 @@
 	box-align: center;
 }
 
+#log .log-message {
+	word-break: break-all !Important;
+	min-width: 100px !Important;
+}
+
+table.grid td.date {
+ word-break: break-all !Important;
```
Comment (MorrisJobke) [inline core/css/mobile.css]: indentation seems to be off

Comment (srt10coupe) [inline core/css/mobile.css]: No, everything is normal as it should be.

Comment (MorrisJobke) [inline core/css/mobile.css]: But it is only indented by 1 space and not by 1 tab like above:
bildschirmfoto von 2015-08-04 13-32-41

Comment (srt10coupe) [inline core/css/mobile.css]: It ist not important

**Comment by srt10coupe** [comment]
Cleaner CSS :+1:

**Comment by jancborchardt** [comment]
@srt10coupe can you please describe what it does and provide a before-after screenshot? cc @owncloud/designers

Also, please squash your commits into one so we have a clean history. Thanks! :)

**Comment by jancborchardt** [comment]
@srt10coupe can you please describe what it does and provide a before-after screenshot?

**Comment by jancborchardt** [comment]
@srt10coupe what’s the matter? :) Why did you just close the issue?

**Comment by srt10coupe** [comment]
Sorry, mistakenly pressed the button.

screenshot_2015-09-16-09-24-58

before

screenshot_2015-09-16-09-23-10

after

the color and shape of the header is not affected.

**Comment by jancborchardt** [comment]
@srt10coupe nice! There seem to be multiple changes in this pull request though. Can you make separate ones, for example one for only improving the log table? (And only one commit for that.) That makes it much easier to review, thanks!

**Comment by jancborchardt** [comment]
If you need help with git or Github, you’re welcome in our IRC channel #owncloud-dev. Just ask there, people will help you. :)

**Comment by srt10coupe** [comment]
when it comes on the weekend, I have professionally just a lot to do. Otherwise very happy :)

**Comment by jancborchardt** [comment]
@srt10coupe any update here? :)

**Comment by PVince81** [comment]
This will need rebasing due to conflicts (and squashing) @srt10coupe

**Comment by jancborchardt** [comment]
Closing because of no update. Please open a new cleaned up pull request when you are done. :)

**Pull request closed without merging**